src/Octoshift/SecretScanningAlerts.cs

```diff
@@ -0,0 +1,23 @@
+namespace OctoshiftCLI
+{
+    public static class SecretScanningAlerts
```

Comment:
Same concern, it's more customary to use singular class name here:
```suggestion
    public static class SecretScanningAlert
```